Tests/iOS-tvOS/SwinjectStoryboardSpec.swift

```diff
@@ -1,200 +0,0 @@
-
-                    let storyboard = SwinjectStoryboard.create(name: "Pages", bundle: bundle, container: container)
-                    let pagesController = storyboard.instantiateInitialViewController() as! AnimalPagesViewController
-                    expect(pagesController.animalPage.hasAnimal(named: "Mimi")) == true
-                }
-            }
-        }
-        describe("Initial view controller") {
-            it("injects dependency definded by initCompleted handler.") {
-                container.storyboardInitCompleted(AnimalViewController.self) { r, c in
-                    c.animal = r.resolve(Animal.self)
-                }
-                container.register(Animal.self) { _ in Cat(name: "Mimi") }
-                
-                let storyboard = SwinjectStoryboard.create(name: "Animals", bundle: bundle, container: container)
-                let animalViewController = storyboard.instantiateInitialViewController() as! AnimalViewController
-                expect(animalViewController.hasAnimal(named: "Mimi")) == true
-            }
-        }
-        describe("AVPlayerViewController") { // Test for Issue #18
-            it("is able to inject a subclass of AVPlayerViewController") {
-                container.storyboardInitCompleted(AnimalPlayerViewController.self) { r, c in
-                    c.animal = r.resolve(Animal.self)
-                }
-                container.register(Animal.self) { _ in Cat(name: "Mimi") }
-                
-                let storyboard = SwinjectStoryboard.create(name: "AnimalPlayerViewController", bundle: bundle, container: container)
-                let animalPlayerViewController = storyboard.instantiateInitialViewController() as! AnimalPlayerViewController
-                expect(animalPlayerViewController.hasAnimal(named: "Mimi")) == true
-            }
-        }
-        describe("Factory method") {
-            it("uses the default shared container if no container is passed.") {
-                SwinjectStoryboard.defaultContainer.storyboardInitCompleted(AnimalViewController.self) { _, _ in }
-                
-                let storyboard = SwinjectStoryboard.create(name: "Animals", bundle: bundle)
-                let animalViewController = storyboard.instantiateViewController(withIdentifier: "AnimalAsCat")
-                expect(animalViewController).notTo(beNil())
-            }
-            
-            afterEach {
-                SwinjectStoryboard.defaultContainer.removeAll()
-            }
-        }
-        // We need to have test bundle deployment target on iOS 9.0 in order to compile storyboards with references.
-        // However, we need to disable these tests when running on iOS <9.0
-        // Using #available(iOS 9.0, *) produces compiler warning for the reasons above
-        if ProcessInfo().isOperatingSystemAtLeast(OperatingSystemVersion(majorVersion: 9, minorVersion: 0, patchVersion: 0)) {
```

Comment:
We can remove the iOS version check because we no longer support iOS 8.

Reply:
Although the current iOS deployment target is set to 12.0, it looks the target was increased by accident in the following PR. It looks good to revert the target to 9.0 later.
https://github.com/Swinject/SwinjectStoryboard/pull/156#pullrequestreview-698432984